### DISCUSSION TITLE
Student Application got rejected

### DISCUSSION THREAD

**Discussion by TenTraicion** [community/community#52752] [GitHub Education] [answered]
Tried 9 times allready
- with valid student id provided by my college
- with valid transcript provided by the college
- with valid registration card
- with my college web portal showing my student dashboard
 it just rejected all the docs instantly without checking anything i suppose.

> given reasons

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

- Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.

first of all, my college is a affilliation college of NU, but if i want to select NU ( as my every single doc's headline is under NU) it told me to use student mail, or mail provided by nu. but NU never provides any student mail. I have my student portal on NU, i can provide live proof though, but the ai didn't gave me a chance to proceed, dunno why. It's really frustrating and annoying even if i have all the document right.

it's also rejected my application while providing student portal ( there is a option called that), and it told me to select other options beside that.

**Comment by Ntshangase** [comment]
Hey @TenTraicion 

Is this your first time applying or you are renewing the student developer pack ?

**Reply by TenTraicion** [reply]
i am renewing

**Reply by Ntshangase** [reply]
ok, If you recently updated your GitHub it's good practice to logout and login again then try to apply again.

**Reply by TenTraicion** [reply]
okay i will try that today and get back to you soon

**Reply by TenTraicion** [reply]
nope, doesn't work

**Reply by Ntshangase** [reply]
From your rejection reasons it seems you have pending assignments in your classroom, Maybe going to global campus and check if there is no pending assignment there that you need to complete which might be stopping you from getting verified.

**Reply by TenTraicion** [reply]
cant access global campus.. and i dont think i have any pending assignments, cause i've completed all last time i got access

**Reply by diyaparmar03966** [reply]
I have also applied with my college ID card, our college is not giving collage email ID. It is really frustrating to apply fo number rof times. System should be fast and simple. GitHub should have practical approach in apprroving student application.

**Comment by sakshgupta** [comment]
> Tried 9 times allready
> 
> * with valid student id provided by my college
> * with valid transcript provided by the college
> * with valid registration card
> * with my college web portal showing my student dashboard
>   it just rejected all the docs instantly without checking anything i suppose.
> 
> > given reasons
> 
> * You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
> * The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
> * Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.
> 
> first of all, my college is a affilliation college of NU, but if i want to select NU ( as my every single doc's headline is under NU) it told me to use student mail, or mail provided by nu. but NU never provides any student mail. I have my student portal on NU, i can provide live proof though, but the ai didn't gave me a chance to proceed, dunno why. It's really frustrating and annoying even if i have all the document right.
> 
> it's also rejected my application while providing student portal ( there is a option called that), and it told me to select other options beside that.

I am sorry to hear about the difficulties you are facing in getting your student identity verified on GitHub. Based on the information you provided, it appears that you have already tried several times to submit your documents for verification, but your applications have been rejected each time with varying reasons.

I understand how frustrating this can be, especially when you have provided all the necessary documents and information. It seems like there might be some confusion regarding the verification process and the requirements for documentation.

One of the reasons for rejection is that your GitHub profile may not be complete with your full name as it appears in your academic affiliation document, and a short bio. I would suggest you make sure that your profile meets these requirements before attempting to verify your identity again.

Regarding the issue with selecting your college affiliation, it appears that there might be some confusion with the selection process. If your college is affiliated with NU, then you should be able to select NU as your affiliation. However, if the option is not available, you may want to reach out to GitHub support for further assistance.

Additionally, I understand that you have provided your student portal information, but it was not accepted as a valid document. If this is the case, I would suggest you consider providing other documents, such as a student ID card, transcript, or registration card, as mentioned in the rejection reasons.

Finally, the suggestion to make more contributions to the GitHub Classroom assignment(s) created by your teacher might be worth considering. This can demonstrate your active participation and involvement in the learning process, which could potentially increase your chances of verification.

 If my response has helped address your concerns and provided helpful information, please consider tagging it as "answered" so that others in the community can easily identify the solution. If you have any further questions or concerns, please do not hesitate to reach out.

**Reply by TenTraicion** [reply]
tried to open a support ticket, but whenever i choose the subject (github student pack) it just redirects me to  https://education.github.com/discount_requests/application

![image](https://user-images.githubusercontent.com/103281314/232054197-803ffdf8-18cd-488e-becb-48daea363490.png)
![image](https://user-images.githubusercontent.com/103281314/232054278-b638148d-a5e7-4feb-bbab-4fda06b85883.png)

**Reply by Ntshangase** [reply]
That's because GitHub no longer supports tickets for Student Developer pack https://github.com/orgs/community/discussions/49366

**Comment by TenTraicion** [comment]
![image](https://user-images.githubusercontent.com/103281314/232052290-f9244e96-932a-433e-abd2-84b9cf754e1a.png)

tried with student id card. registration card, academic transcript and none works.
about school affiliation tried checking NU but alas, it just dont give any option to proceed to next step and tells me to try with a email address provided by nu, which nu never gives.

about the contribution on assignment given my teacher, alas none of my teacher is in github, i just practice by my self and contribute on open source, so its a no no for me. 

it's just a headache to me right now. guess thats what happens after ms took over.

**Reply by sakshgupta** [reply]
> ![image](https://user-images.githubusercontent.com/103281314/232052290-f9244e96-932a-433e-abd2-84b9cf754e1a.png)
> 
> tried with student id card. registration card, academic transcript and none works. about school affiliation tried checking NU but alas, it just dont give any option to proceed to next step and tells me to try with a email address provided by nu, which nu never gives.
> 
> about the contribution on assignment given my teacher, alas none of my teacher is in github, i just practice by my self and contribute on open source, so its a no no for me.
> 
> it's just a headache to me right now. guess thats what happens after ms took over.

Unfortunately, a college email ID is a requirement for the program, and without it, it's not possible to avail of the benefits of the pack. As for your contribution to the assignment given by your teacher, it's great that you're practicing by yourself and contributing to open-source projects. However, if you have any more questions or issues, please feel free to contact me. Also, I kindly request you to mark this issue as answered, as it will help me too.

**Reply by TenTraicion** [reply]
how come i get a college email id if my college never provided one? that's just ridiculous. anyway thanks for the help, but the issue isn't solved yet.

**Reply by adityansx** [reply]
I am facing the same issue as well. It's frustrating as my college does not provide us with a mail ID. And each time I apply for re-verification, it rejects my application because I don't have a college provided mail ID.

**Comment by TenTraicion** [comment]
Dunno what to do after this:
![image](https://user-images.githubusercontent.com/103281314/233855638-d6034baa-c708-4d82-9382-c7d729a28e7c.png)

**Comment by TenTraicion** [comment, marked as answer]
Note: Github recently update the application system. Now if you have added your school email to your account, github will automatically fetch your school for you. and then you will only have to upload a verification document. That's all. After submission, wait for 12-72 hours and your student benefits will be added to your account. Really appreciated this update from github.

**Reply by raba13** [reply]
I applied with my university student email, image of my Uni ID and every other information in the form on 17/03/2024 (NZ), and awaiting for the Copilot benefits added to my account.
Thanks

**Comment by Kiran-Naseer** [comment]
Dear GitHub Support Team,

I'm reaching out for help with creating a new account. Despite multiple attempts, I'm facing issues with the verification process:

1. ID Card Photo: When I upload my ID card photo, it's not being accepted. Can you please clarify what's the issue?
2. Transcript: When I upload my transcript, you're saying it only shows a logo. What else is required? 
Please assist me in resolving these issues so I can create my new account. I'd appreciate your guidance on what I need to do.

![Screenshot_20250504_200045_Gmail](https://github.com/user-attachments/assets/457e758a-3083-47ad-ba72-476809fd489d)
